src/main/java/Containers/Portfolio.java

```diff
@@ -23,67 +23,116 @@ public static Portfolio getInstance() {
         return Portfolio.commonPortfolio;
     }
 
-    // Storage: The UUID of asset and the reference of asset
-    private final HashMap<String, Asset> storage;
+
+
+    // assetList: a list of asset in the portfolio
+    private List<Asset> assetList;
+    // transactionList: a list of transaction in the portfolio
+    private List<Transaction> transactionList;
+    // votingHistory: a history of votes in the portfolio
+    private List<Vote> votingHistory;
+    private double profitability;
```

Comment:
How is this measured? Is this the net profit? Consider renaming variable to be more concise.

Reply:
This is related to the voting power algorithm, which we haven't decided yet.